docs/how-to/logging.rst

```diff
@@ -0,0 +1,235 @@
+.. meta::
+   :description: HIP provides a logging mechanism that allows you to trace HIP API and runtime codes
+   when running a HIP application.
+   :keywords: AMD, ROCm, HIP, logging
```

Comment:
@samjwu  can you help deal with common keywords centrally?

Reply:
as in always insert AMD and ROCm along with a meta directive?

Reply:
Yes, AMD, ROCm and the project name. Let's start with those three